configs/PicoFightingBoard/README.md

```diff
@@ -0,0 +1,5 @@
+# GP2040 Configuration for Pico Fighting Board
+
+![Pin Mapping](assets/PicoFightingBoard_v1.1.jpg)
+
+Open source board based on the Brook board design: <https://github.com/FeralAI/PicoFightingBoard>. The board labels are associated with the GP2040 generic labeling. The 20-pin connector, 4-pin connector and screw terminals are button-compatible with the Brook wiring harnesses and button mappings.
```

Comment:
Please update this to note that it has been adapted from the new GP2040-CE standard layout.